email/1.7/modules/ROOT/pages/email-gmail.adoc

```diff
@@ -3,13 +3,11 @@
 
 To connect Anypoint Connector for Email (Email Connector) to your Gmail account, enable *Less Secure Apps* in your account and configure a global element for the server connection IMAPS, POP3S, or SMTPS you want to use.
 
-== Enable Less Secure Apps in Your Gmail Account
+== Configure an App Password in Your Gmail Account
 
-Google might block third-party mail clients if you do not enable the *Less Secure Apps* setting for your Gmail account, because Google blocks any apps that are not using security protocols that Google deems mandatory unless you allow less secure apps to access your Gmail account.
+In order to send emails to a Gmail account via the Email Connector, you need to xref:https://support.google.com/accounts/answer/185833?visit_id=637917932546436561-2352829986&p=InvalidSecondFactor&rd=1[configure an app password] in your Gmail account.
 
-When you try to connect to your Gmail for the first time, you will receive an email with a link for enabling less secure apps in your Gmail account.
-
-image::email-gmail-security.jpg[Email notification from Google about the blocked sign-in attempt,height=389,width=421]
+Use the app password instead of your regular Gmail email password, which enables you to use email protected by MFA (Multi-Factor Authentication)
```

Comment:
```suggestion
Use the app password instead of your regular Gmail email password, which enables you to use email protected by MFA (Multi-Factor Authentication).
```